app/views/download/package.erb

```diff
@@ -69,10 +69,13 @@ pacman -Sy <%= repo_name %>/<%= @package %></pre>
                   <pre><%=
                      case v[:flavor]
                      when 'openSUSE', 'SLE'
+                       if @project != "openSUSE:Factory" and @project != "openSUSE:Tumbleweed" and not @project .start_with? "openSUSE:Leap:"
```

Comment:
I'm not sure if this list contains all projects we need to filter for Leap 15.3 since it added additional official repos

Reply:
15.3 does make this a bit more difficult to catch, considering unlike previously, there is no standard prefix to really glob here. That being said, globbing using existing project names should work just fine